### DISCUSSION TITLE
[Feature] Order "people" icons by latest photo date added to person

### DISCUSSION THREAD

**Discussion by EnriqueGF** [immich-app/immich#5027] [Feature Request]
### The feature

Hello,

When we enter on "Explore", we do have a "People" label following by the icons of people.

We could order this by the latest added photos to people, it means, if Rob has 5.000 photos and 1 was added just now, it would go the first.

I would find this really useful to quickly find people that you uploaded recently photos of.

And it's a simple change I think

Thank you.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by Gamrin** [comment]
As an additional, maybe make it possible to set a static image for people
